spec/models/user_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
- [ ] Please make sure to add unit and integrations tests for your app as it is a hard requirement in this project.

Reply:
Thanks for this I will implement that

Reply:
You are welcome ❤️